ui/src/ifql/components/FuncArg.tsx

```diff
@@ -13,7 +13,7 @@ interface Props {
   funcName: string
   funcID: string
   argKey: string
-  value: string | boolean
+  value: string | boolean | {[x: string]: string}
```

Comment:
In lines 108 and 116 value is used in a div as the value, if it is an object, won't that cause issues?

Also, it looks like in all the other places that value is used in this file are through the getters `value()` and `boolValue()`. Should these functions take into account the possibility of `value` being an object?

Reply:
This part of the IFQL builder can't handle objects yet